src/c_api.cpp

```diff
@@ -421,7 +421,7 @@ class Booster {
     }
 
     return Predictor(boosting_.get(), start_iteration, num_iteration, is_raw_score, is_predict_leaf, predict_contrib,
-                        config.pred_early_stop, config.pred_early_stop_freq, config.pred_early_stop_margin);
```

Comment:
could this whitespace-only change be a separate, quick PR? Just to remove this file from the diff?

I actually think it's informative to see a PR like this and be able to say "`c_api.cpp` remained unchanged", since that file specifically defines the public API of LightGBM.

Reply:
and this one in `gbdt.h`

![image](https://user-images.githubusercontent.com/7608904/135702336-d6b69175-802d-439a-91e7-eb5706db4929.png)